desktop_version/CMakeLists.txt

```
@@ -169,6 +174,25 @@ endif()
 set(XML2_SRC
     ../third_party/tinyxml2/tinyxml2.cpp
 )
+set(FAUDIO_SRC
+    ../third_party/FAudio/src/F3DAudio.c
+	../third_party/FAudio/src/FACT3D.c
+	../third_party/FAudio/src/FACT.c
+	../third_party/FAudio/src/FACT_internal.c
+	../third_party/FAudio/src/FAPOBase.c
+	../third_party/FAudio/src/FAPOFX.c
+	../third_party/FAudio/src/FAPOFX_echo.c
+	../third_party/FAudio/src/FAPOFX_eq.c
+	../third_party/FAudio/src/FAPOFX_masteringlimiter.c
+	../third_party/FAudio/src/FAPOFX_reverb.c
+	../third_party/FAudio/src/FAudio.c
+	../third_party/FAudio/src/FAudioFX_reverb.c
+	../third_party/FAudio/src/FAudioFX_volumemeter.c
+	../third_party/FAudio/src/FAudio_internal.c
+	../third_party/FAudio/src/FAudio_internal_simd.c
+	../third_party/FAudio/src/FAudio_operationset.c
+	../third_party/FAudio/src/FAudio_platform_sdl2.c
```

Here's the juicy part of why I wanted to statically link:

```suggestion
	../third_party/FAudio/src/FAPOBase.c
	../third_party/FAudio/src/FAudio.c
	../third_party/FAudio/src/FAudio_internal.c
	../third_party/FAudio/src/FAudio_internal_simd.c
	../third_party/FAudio/src/FAudio_operationset.c
	../third_party/FAudio/src/FAudio_platform_sdl2.c
```

---

Removed the unneeded, thanks! Didn't know which ones were the unneeded ones